src/elements/loading-indicator/loading-indicator.stories.ts

```diff
@@ -74,79 +62,21 @@ const color: InputType = {
 };
 
 const defaultArgTypes: ArgTypes = {
-  variant,
   size,
   color,
 };
 
 const defaultArgs: Args = {
-  variant: variant.options![0],
   size: size.options![0],
   color: color.options![0],
 };
 
-export const WindowSmallDefault: StoryObj = {
+export const Default: StoryObj = {
   render: Template,
   argTypes: defaultArgTypes,
   args: { ...defaultArgs },
 };
```

Comment:
For consistency with the `circle` variant, i would also add the color variant stories (maybe one for the size too)

Reply:
It was me deleting all the stories, but it would be ok also for me having one size and one color story. But I think it would be also ok to have only one story :)

Reply:
I'm not against the single story but, in that case, I would have a single story for the circle too

Reply:
Ok would also be ok. But there there is only one prop, so it does not explode by providing all color variants :D